stable_baselines3/common/logger.py

```diff
@@ -300,13 +316,23 @@ def write(self, key_values: Dict[str, Any], key_excluded: Dict[str, Union[str, T
             if isinstance(value, Video):
                 raise FormatUnsupportedError(["csv"], "video")
 
-            if isinstance(value, Figure):
+            elif isinstance(value, Figure):
                 raise FormatUnsupportedError(["csv"], "figure")
 
-            if isinstance(value, Image):
+            elif isinstance(value, Image):
                 raise FormatUnsupportedError(["csv"], "image")
 
-            if value is not None:
+            elif isinstance(value, str):
+                # escape any funky characters that may break the csv
+                value = escape_unicode(value)
+
+                # escape quotechars by prepending them with another quotechar
+                value = value.replace(self.quotechar, self.quotechar + self.quotechar)
```

Comment:
this is new, no (I saw only the escape unicode before)? is that tested?

Reply:
I tested it by hand and in automatic testing it should be covered by test_logger.py:read_log ? I added it in 474ce91344558b1f21da7a06ec3dcf17563eb616

Reply:
but without the quotechar escape, shouldn't the test fail? (or it is just for better visualization?)
because your tests passed before that addition, no?

Reply:
Just realized that the test passed because the test input doesn't contain a quotechar (duh). I'll add it.

Reply:
Added the new test input in 8dbe360b84a88cadbae7ee6fb319aec823ffdf53